app/components/projects/item/component.tsx

```diff
@@ -0,0 +1,92 @@
+import React from 'react';
+import cx from 'classnames';
+
+import Avatar from 'components/avatar';
+import Button from 'components/button';
+
+export interface ItemProps {
+  id: string;
+  className?: string;
+  name: string;
+  area: string;
+  description: string;
+  lastUpdate: string;
+  contributors?: Record<string, unknown>[];
+  onDownload: (event: React.MouseEvent<HTMLButtonElement>) => void | unknown;
+  onDuplicate: (event: React.MouseEvent<HTMLButtonElement>) => void | unknown;
+  onDelete: (event: React.MouseEvent<HTMLButtonElement>) => void | unknown;
+}
+
+export const Item: React.FC<ItemProps> = ({
+  className,
+  name,
+  area,
```

Comment:
I could buy the `areaName` thing but as we are inside the project `Item` I think it's redundant to say `projectName`. Because we should do the same with description, contributors, etc...